repos/system_upgrade/el7toel8/actors/setpermissiveselinux/actor.py

```diff
@@ -25,10 +25,11 @@ def process(self):
                 success, err_msg = selinux_set_permissive()
                 if not success:
                     # FIXME: add an "action required" flag later
-                    report_with_remediation(
-                        title='Could not set SElinux into permissive mode',
-                        summary='{}'.format(err_msg),
-                        remediation='Please set SElinux into permissive mode manually.',
-                        severity='high',
-                    )
+                    create_report([
+                        reporting.Title('Could not set SElinux into permissive mode'),
+                        reporting.Summary('{}'.format(err_msg)),
+                        reporting.Severity(reporting.Severity.HIGH),
+                        reporting.Tags([reporting.Tags.SELINUX, reporting.Tags.SECURITY]),
+                        reporting.Flags([reporting.Flags.FAILURE, reporting.Flags.INHIBITOR])
```

Comment:
again inhibitor in finalization phase

Reply:
yep, did not realized that back then. I think I made the same brain-fart even during the first reporting phase :)